src/Tabs/TabTemplate.spec.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-env mocha */
+import React from 'react';
+import {shallow} from 'enzyme';
+import {expect} from 'chai';
+import TabTemplate from './TabTemplate';
+import getMuiTheme from '../styles/getMuiTheme';
+
+describe('<TabTemplate />', () => {
+  const muiTheme = getMuiTheme();
+  const shallowWithContext = (node) => shallow(node, {context: {muiTheme}});
+
+  it('should have different tab template style', () => {
+    const wrapper = shallowWithContext(
+      <TabTemplate
+        style={{backgroundColor: 'red'}}
+        selected={false}
+      />
+    );
+
+    expect(wrapper.prop('style')).to.have.property('backgroundColor', 'red');
```

Comment:
We use `assert.strictEqual` everywhere else on the code base.
Why? Because the API is much simpler and conceptually simpler to understand for new contributors.